big-structure/src/destress_big_structure/console.py

```diff
@@ -387,97 +375,132 @@ def headless_destress(pdb_file: str) -> DesignMetricsOutputRow:
         "aggrescan3d_max_value",
     ]
 
+    # Loading in the PDB file and converting it to an ampal assembly
     try:
+        ampal_assembly = ampal.load_pdb(str(pdb_file), path=True)
 
-        # Running the DE-STRESS metrics for the pdb file
-        design_metrics = analysis.create_metrics_from_pdb(pdb_string_filtered)
+    except ValueError:
+        print("PDB file could not be loaded due to a ValueError.")
```

Comment:
The value error isn't propagated here, you might want to do something like
```python
except ValueError as e:
    print(f"PDB file could not be loaded due to a ValueError:\n {e}")

Reply:
This change has been made .